jdaviz/configs/specviz/helper.py

```diff
@@ -229,6 +229,28 @@ def _redshift_listener(self, msg):
         if msg.param == "redshift":
             self._redshift = msg.value
 
+    def activate_scientific_notation(self, activate=True, axis=None):
```

Comment:
This still hardcodes the formatting, in a sense. I wonder if this can be generalized. 
```suggestion
    def set_spectrum_tick_format(self, fmt, axis=None):
```